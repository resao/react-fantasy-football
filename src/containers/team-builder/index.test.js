import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import { TeamBuilder } from "./";

configure({ adapter: new Adapter() });

describe("<TeamBuilder/>", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<TeamBuilder onInitPlayers={() => {}} />);
  });

  it("should match snapshot when receiving players", () => {
    wrapper.setProps({
      players: {
        goalkeeper: 1,
        defender: 3,
        midfielder: 5,
        forward: 2,
      },
    });

    expect(wrapper).toMatchSnapshot();
  });
});
