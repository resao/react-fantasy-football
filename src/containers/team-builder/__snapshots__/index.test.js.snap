// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TeamBuilder/> should match snapshot when receiving players 1`] = `
<Fragment>
  <modal
    modalClosed={[Function]}
    show={false}
  >
    <_default
      players={
        Object {
          "defender": 3,
          "forward": 2,
          "goalkeeper": 1,
          "midfielder": 5,
        }
      }
      saveCancelled={[Function]}
      saveContinued={[Function]}
    />
  </modal>
  <team
    players={
      Object {
        "defender": 3,
        "forward": 2,
        "goalkeeper": 1,
        "midfielder": 5,
      }
    }
  />
  <controls
    players={
      Object {
        "defender": false,
        "forward": false,
        "goalkeeper": false,
        "midfielder": false,
      }
    }
    saved={[Function]}
    valid={true}
  />
</Fragment>
`;
